src/server/ocpi/ocpi-services-impl/ocpi-2.1.1/OCPIUtilsService.ts

```diff
@@ -226,7 +226,8 @@ export default class OCPIUtilsService {
                   period_begin: '08:00',
                   period_end: '18:00'
                 },
-              ]
+              ],
+              twentyfourseven: false
```

Comment:
Weird, the doc says 'Choice: one of two'

![image](https://user-images.githubusercontent.com/25266945/117728248-7e561880-b1e9-11eb-9006-042d832ef727.png)